player.py

```diff
@@ -84,51 +84,29 @@ def score_hand(self, anthill):
     for food in self.hand:
       self.score += anthill.index(K_FOOD_V_COLOUR_DICT[food]) * self.hand[food]
 
-  def choose_ant_to_move(self, allowed_choices):
-    """Player chooses which colour of insect meeple to move this turn
+  def make_choice(self, allowed_choices, prompt_text):
```

Comment:
It seems more intuitive for me to place the `prompt_text` before the `allowed choices` but that's a personal decision. 

Additionally, this could be a good opportunity to learn about and practice using keyword arguments with default values. For example, `prompt_text='choose between'`. However, upon looking at the contents of the function, I think this would make more sense once the `allowed_choices` are incorporated into the input text.